design/design.md

```diff
@@ -23,3 +23,13 @@
 - We did not change the UI design of the app from lab 2.
```

Comment:
There are 1600 files that have changed in this PR. That's too many for me to look at in the Files Changed view.

I understand that you removed the ```node_modules directory```, which is good, but that should have been done outside of this pull request because of this very problem of too many files.

My suggestion for a fix:
* Create a new branch based on ```lab3``` called ``` lab3revised```
* In that new branch, remove ```node_modules```
* Make a branch from that ```lab3revised``` named ```lab4revised````
* Move the files from ```lab4``` that you've changed into ```lab4revised```
* Commit those changes and create a PR from ```lab4revised``` to ```lab3smaller```
* Verify the new PR has many fewer files (and contains all the changes you want)

Let me know when you've got a new PR I can review.

Thanks,

Prof. Rhodes